docs/gethue/content/en/posts/2021-04-09-collaborate-on-your-sql-queries-and-results-directly-within-slack.md

```diff
@@ -0,0 +1,122 @@
+
+To manually set-up the Hue App, follow these steps mentioned in the [Hue docs](https://docs.gethue.com/administrator/configuration/server/#manual-slack-app-installation)
+
+Now, open Hue, run some query and copy its link:
+
+![Run Query in Hue](https://cdn.gethue.com/uploads/2021/04/run_query_in_hue.png)
+
+Paste it in the Slack channel and see the unfurl magic!
+
+![Query Link Preview](https://cdn.gethue.com/uploads/2021/04/query_link_preview.png)
+
+### Wonder why result is shown like this?
+
+Slack currently does not support markdown tables and potential improvements with inline preview will come when Hue supports result caching via [query tasks](https://docs.gethue.com/administrator/administration/reference/#task-server).
+
+After evaluating a lot of possible fixes (like uploading result image, truncating columns which doesn't look good, pivoting table, [uploading result file](https://github.com/slackapi/python-slack-sdk/issues/991) etc.) and seeing their tradeoffs, we chose to have few sample rows but keep all columns by pivoting the result table and to compensate for the loss of rows, Hue app gives the result file in the message thread.
+
+![Message Thread with Result File](https://cdn.gethue.com/uploads/2021/04/message_thread_with_result_file.png)
+
+Users can share the SQL gists too!
+
+![Gist Link](https://cdn.gethue.com/uploads/2021/04/gist_link.png)
+
+![Gist Link Preview](https://cdn.gethue.com/uploads/2021/04/gist_link_preview.png)
+
+### Read access
+
+Keeping in mind the security aspect, those Slack users who are Hue users and have the read permissions to access the query and its result will get this rich preview and result file after sharing the link. This mapping is currently done by checking the email prefix and its host for Hue username.
+
+For example, some person ‘Alice’ having a Hue account with username ‘alice’ can have read access from some Slack account only if the email prefix of that slack user is same and Hue username and the email host is same in the Hue domain i.e. **alice@gethue.com slack user** can only access **Hue user ‘alice’** on **‘hue.gethue.com’**.
+
+### What's coming up next?
+
+In the follow-ups, promoting the Share to Slack from the Hue side where users can select the channel to send the link.
+
+Also stay tuned for more such functionalities such as replying to users asking questions on how to find certain data tables or to query them and also send results for scheduled queries in certain Slack channels!
+
+### Try it out!
+
+Get your hands on this beta feature by:
+
+Running some queries on the [demo live instance](https://demo.gethue.com/).
+
+Joining the [demo Slack workspace](https://join.slack.com/t/hue-bot-dev/shared_invite/zt-opqwvv68-eQFeios8FzFbmqQJ5wBNzg) to see the app in action or by using the following **demo Slack account credentials** to share some query links in the **demo workspace**.
```

Comment:
`**demo Slack account credentials** to share some query links in the **demo workspace**.` --> credentials